core/src/main/scala/org/apache/spark/metrics/MetricsSystem.scala

```diff
@@ -133,7 +133,11 @@ private[spark] class MetricsSystem private (
 
     if (instance == "driver" || instance == "executor") {
       if (metricsNamespace.isDefined && executorId.isDefined) {
-        MetricRegistry.name(metricsNamespace.get, executorId.get, source.sourceName)
+        if (source.sourceName.contains("TIBCO ComputeDB")) {
```

Comment:
What name it will use if product is OSS (i.e SnappyData)?

Reply:
Added code change to provide namespace based on product i.e enterprise or OSS